README.md

```diff
@@ -305,6 +306,12 @@ Everyone interacting with this project is expected to follow the
 Should you discover any security issues, please refer to sigstore's [security
 process](https://github.com/sigstore/.github/blob/main/SECURITY.md).
 
+### SLSA Provenance
+This project emits a SLSA provenance on it's release! This enables you to verify the integrity
+of the downloaded artifacts and ensured that the binary's code really comes from this source code.
+To do so, please follow the instructions [here](https://github.com/urllib3/urllib3/blob/main/.github/workflows/publish.yml).
```

Comment:
I don't think we should link to another project's workflow here, but I'm also not seeing instructions here on how to verify integrity.

Reply:
Agreed. Is there an official (and human-first) doc we can link to instead, @diogoteles08?

Reply:
Ups, good catch! It was my mistake, for some reason I put the wrong URL. It was supposed to be this one: https://github.com/slsa-framework/slsa-github-generator#verification-of-provenance
If that makes sense now, please react and I'll commit the change

Reply:
That looks good, thanks!

Reply:
I think that's fine for now.